tests/integration/backward_compatible/dbapi20.py

```diff
@@ -0,0 +1,831 @@
+#!/usr/bin/env python
+""" Python DB API 2.0 driver compliance unit test suite. 
+    
+    This software is Public Domain and may be used without restrictions.
+
+ "Now we have booze and barflies entering the discussion, plus rumours of
+  DBAs on drugs... and I won't tell you what flashes through my mind each
+  time I read the subject line with 'Anal Compliance' in it.  All around
+  this is turning out to be a thoroughly unwholesome unit test."
+
+    -- Ian Bicking
+"""
+
+__version__ = "1.15.0"
+
+import unittest
+import time
+import sys
+
+if sys.version[0] >= "3":  # python 3.x
```

Comment:
do we need those?

Reply:
I've copied that file verbatim from the original source. I think we can keep them unless they are causing issues.